### DISCUSSION TITLE
2022.11.19

### DISCUSSION THREAD

**Discussion by itmakesmesoft** [itmakesmesoft/Steady-Study#156] [Self-Assessment]
이번 주 목표를 이루기 위해 어떤 노력을 했는지 스스로 평가해보고, 점수를 매겨주세요.
ex)
이번 주 목표 : 기술 블로그에 배운 내용 정리하기

점수 : 80점
이유 : (생략)

**Comment by giokim12** [comment]
김지오 : 블로그 글 80개 채우기, 생활코딩 리액트 완강, 코딩애플 뷰 part1
점수: 0점,,,
 
변명을 살짝 하자면,,, 그만큼 이제 관통 플젝에 시간을 많이 썼다~~ 이겁니다...
네... 앞으로는 제 분수를 잘 아는 사람이 될게요,,,
